src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -6720,6 +6720,42 @@ def test_aks_create_with_azuremonitormetrics(self, resource_group, resource_grou
             self.is_empty(),
         ])
 
+    @AllowLargeResponse(8192)
```

Comment:
As this test will download the cli extension k8s-extension, resulting in a larger size of the corresponding recording file, I recommend marking this case as `@live_only()`, so that CI will not execute the test, but the cli runner (hosted by AKS team) will continue to ensure the quality of the case

Reply:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=81287571&view=results) to validate the change, test passed